packages/edit-navigation/src/components/layout/use-menu-notifications.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+
+export default function useMenuNotifications( menuId ) {
+	const { lastSaveError, lastDeleteError } = useSelect( ( select ) => ( {
+		lastSaveError: select( 'core' ).getLastEntitySaveError(
+			'root',
+			'menu'
+		),
+		lastDeleteError: select( 'core' ).getLastEntityDeleteError(
+			'root',
+			'menu',
+			menuId
+		),
+	} ) );
```

Comment:
Shouldn't we want to include `[menuId]` as the second argument of `useSelect`?

Reply:
d02bcb8c86